app/main.py

```diff
@@ -2,5 +2,13 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args: tuple) -> [list, int]:
```

Comment:
No need to add type annotations to *args. You can use, for example, `*args: int` if you want to indicate that func expects integer arguments